### PR TITLE
Calculator UI

### PR DESCRIPTION
**Pull request by tylher**
- [x] Build user interface for calculator
- [x] style user interface

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,23 +1,10 @@
...
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div>
+      <Calculator />
     </div>
   );
 }
```
Comment (elisha2kyakpo1) [inline src/App.js]: - [x] Kindly make this component (App.js) a class-based component as well.

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @tylher,

You have done a great job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

- The PR title and description look professional 👍
- The commit messages are descriptive 👍
- No linter errors 👍

## Required Changes ♻️
_Check the comments under the review._

- [x] Kindly customize the readme file and include all the necessary commands.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by elyor-doniyorov — APPROVED** [review]
Hi @tylher, 

In my opinion, your project milestone is complete!  There is nothing else to say other than... well done :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Comment by tylher** [comment]
Thank you @elyor-doniyorov for the approval

**Pull request merged**